tools/generator/Tests-Core/expected.cp/Xamarin.Test.Invalidnames.InvalidNameMembers.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using Android.Runtime;
+
+namespace Xamarin.Test.Invalidnames {
+
+	// Metadata.xml XPath class reference: path="/api/package[@name='xamarin.test.invalidnames']/class[@name='InvalidNameMembers']"
+	[global::Android.Runtime.Register ("xamarin/test/invalidnames/InvalidNameMembers", DoNotGenerateAcw=true)]
+	public partial class InvalidNameMembers : Java.Lang.Object {
+
+
+		// Metadata.xml XPath field reference: path="/api/package[@name='xamarin.test.invalidnames']/class[@name='InvalidNameMembers']/field[@name='INVALID_$CONSTANT']"
+		[Register ("INVALID_$CONSTANT")]
+		public const int Invalid$constant = (int) 1;
+
+		static IntPtr INVALID_$FIELD_jfieldId;
+
+		// Metadata.xml XPath field reference: path="/api/package[@name='xamarin.test.invalidnames']/class[@name='InvalidNameMembers']/field[@name='INVALID_$FIELD']"
+		[Register ("INVALID_$FIELD")]
```

Comment:
huh, yeah it doesn't make sense. Why does tests pass even with this? The answer is most likely because "unless I run 'make clean' it does not run tests against the binaries I just built"...